src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1740,6 +1736,7 @@ export const ComputeModalBase = ({
               div({ style: { marginTop: '0.5rem' } }, [
                 h(NumberInput, {
                   id,
+                  'aria-labelledby': id,
```

Comment:
You are setting the ID of the label to itself, which does not make sense.

![image](https://user-images.githubusercontent.com/484484/192605250-097bee41-3a09-43cb-822a-a9776681a098.png)

Reply:
This can be removed, added it when I was confused at one point from a test suite warning (that has since been resolved properly)

Reply:
I'm still seeing an issue in that modal, but created a ticket for it because it's not related to your PR. https://broadworkbench.atlassian.net/browse/IA-3767